packages/api/src/controllers/customAnimalTypeController.js

```diff
@@ -30,6 +33,43 @@ const customAnimalTypeController = {
       }
     };
   },
+
+  addCustomAnimalType() {
+    return async (req, res) => {
+      const trx = await transaction.start(Model.knex());
+      try {
+        const { farm_id } = req.headers;
+        const { type } = req.body;
```

Comment:
Maybe we don't need this... but what do you think about trimming strings for trailing spaces? We could do it on the FE but we do have the API technically :p .

Reply:
I like that as a rule but not necessarily as a one-off; I think translation strings were the only place we had been running `trim()` on user input so far, so I could see it as replacement for that in these tables that don't have the translation string column.

Were you thinking a new guideline or something more programatic like a change to `postWithResponse()`?

Reply:
Uhm, I think I was thinking like Gursimrans function formatTranslationKey() as some some of update to our new style. Up to you but it seems possibly quite useful.